config/jobs/cert-manager/cert-manager/release-1.8/cert-manager-1.8.yaml

```diff
@@ -324,48 +317,9 @@ presubmits:
           value: "1"
     branches:
     - release-1.8
-    - release-1.9
     always_run: true
     optional: false
-  - name: pull-cert-manager-e2e-v1-24-upgrade
-    max_concurrency: 4
-    agent: kubernetes
-    decorate: true
-    annotations:
-      description: Runs cert-manager upgrade from latest published release
-    labels:
-      preset-default-e2e-volumes: "true"
-      preset-dind-enabled: "true"
-      preset-make-volumes: "true"
-      preset-service-account: "true"
-    spec:
-      containers:
-      - image: eu.gcr.io/jetstack-build-infra-images/bazelbuild:20220512-b6ea825-4.2.1
-        args:
-        - runner
-        - make
-        - K8S_VERSION=1.24
-        - vendor-go
-        - test-upgrade
-        resources:
-          requests:
-            cpu: 3500m
-            memory: 12Gi
-        securityContext:
-          privileged: true
-          capabilities:
-            add:
-            - SYS_ADMIN
-      dnsConfig:
-        options:
-        - name: ndots
-          value: "1"
-    branches:
-    - release-1.8
-    - release-1.9
-    always_run: true
-    optional: false
```

Comment:
note: this test shouldn't run against release-1.8, so its removal is actually a bugfix of an issue which currently blocks backporting for release-1.8